### ISSUE TITLE
Request: User config file

### ISSUE THREAD

**Issue by orschiro** [xypiie/spread0r#16]
Hi there, 

Is it possible to add support for a user config file to store preferences such as the default wpm?

Currently I have to reapply my preferred settings on each startup of spread0r.

Thanks!

**Comment by goiken** [comment]
What else besides the wpm would you like to configure? Because if it were only for the wpm, I would just [edit the code](https://github.com/xypiie/spread0r/blob/master/spread0r.pl#L43) to your likings.

**Comment by wese3112** [comment]
Maybe the program could remember what files the user read and where he stopped reading. With this you could start reading a file, stop and later continue from there. I know there's the option to 'scroll' through the sentences, but if you are at e.g. sentence 400, close the program and want to continue some days later, you'd have to klick the 'next sentence' buttons 400 times before you can continue.

short version: Maybe a config-file-like file could be used to remember the files you read and where you paused reading :)

**Comment by orschiro** [comment]
@wese3112 

This is a really cool idea!

Yes, essentially just a file to remember where you stopped reading last time and which wpm you have been using.
